papaparse.js

```diff
@@ -269,7 +269,25 @@
 		var _newline = '\r\n';
 
 		/** quote character */
-		var _quoteChar = '"';
+// lunatixcoer: Changed logic so you can ignore quote characters altogether.
```

Comment:
I will prefer to have a simpler comment. Like: "Use default value only if it's not set".